post/templates/post/index.html

```diff
@@ -68,7 +68,13 @@
 
                                                 <!-- Read more: for long post -->
                                                 <a target="_blank" href="{% url 'post:detail' post.id %}">Read More</a>
-                                                <div><img src="#"></div>
+                                                <div>
+                                                    {% if post.image %}
+                                                        <img class="media-object" style="width: 48px; height: 48px;"
```

Comment:
Minor style issue but I recommend moving in-line styles into our `style.css` and giving it a nice declarative title so it's immediately clear what the styling is supposed to do. :)